infinite_base.module

```diff
@@ -242,7 +242,7 @@ function infinite_base_get_all_parents($term) {
  * Implements hook_preprocess_node().
  */
 function infinite_base_preprocess_node(&$variables) {
-  if ($variables['view_mode'] == 'full' || $variables['view_mode'] == 'lazyloading') {
+  if ($variables['view_mode'] == 'full' || $variables['view_mode'] == 'lazyloading' && \Drupal::moduleHandler()->moduleExists('ad_integration')) {
```

Comment:
this check is wrong, the "&&" operator is binding stronger then the "||" operator, which results in a faulty check, it should be something like this (see additional parenthesis arround the "||" operands):
`if ( ($variables['view_mode'] == 'full' || $variables['view_mode'] == 'lazyloading') && \Drupal::moduleHandler()->moduleExists('ad_integration'))`